### ISSUE TITLE
NPE due to a passed through an AsyncTask

### ISSUE THREAD

**Issue by DavidEdwards** [googlecast/CastCompanionLibrary-android#294]
[Probable cause 1](https://github.com/googlecast/CastCompanionLibrary-android/blob/master/src/com/google/android/libraries/cast/companionlibrary/cast/VideoCastManager.java#L2450)

[Probable cause 2](https://github.com/googlecast/CastCompanionLibrary-android/blob/master/src/com/google/android/libraries/cast/companionlibrary/cast/VideoCastManager.java#L2349)

It looks like `bitmap` needs a null check.

```
Fatal Exception: java.lang.NullPointerException
       at android.graphics.Bitmap.copy(Bitmap.java:558)
       at android.support.v4.media.session.MediaSessionCompat$MediaSessionImplBase.cloneMetadataIfNeeded(SourceFile:5296)
       at android.support.v4.media.session.MediaSessionCompat.setMetadata(SourceFile:404)
       at com.google.android.libraries.cast.companionlibrary.cast.VideoCastManager$29.onPostExecute(SourceFile:3349)
       at android.os.AsyncTask.finish(AsyncTask.java:632)
       at android.os.AsyncTask.access$600(AsyncTask.java:177)
```

**Comment by naddaf** [comment]
Addressed in v2.8.4

**Comment by ankitagrawal2411** [comment]
this is a bug in support library v4 itself basically it does copy of bitmap without checking whether the bitmap.getConfig() is null or not , if it null this exception is thrown , i have filed a bug https://code.google.com/p/android/issues/detail?can=2&start=0&num=100&q=&colspec=ID%20Status%20Priority%20Owner%20Summary%20Stars%20Reporter%20Opened&groupby=&sort=&id=221795
